common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -414,6 +414,12 @@ div.problem {
     }
   }
 
+  div.inline {
+    > span {
```

Comment:
@jolyonb - I was asking if you could change your class on the div to "wrapper-inline" since just "inline" is more common and not entirely logical since the inline elements in your example are inside the inline class. "wrapper-inline" is more accurate and easier to follow since it wraps the inline elements.